### ISSUE TITLE
Update favicons-webpack-plugin dependency

### ISSUE THREAD

**Issue by peacememories** [dillonkearns/elm-pages#32]
Currently I am not able to use elm-pages on macos with node 13, because the version of `favicons-webpack-plugin` it depends on in turn depends on `sharp` v0.22.1. The first version supporting node 13 is v0.23.2, which is depended on by the current version of `favicons-webpack-plugin`, so a dependency update should fix this.

**Comment by dillonkearns** [comment]
Hey, thanks for the suggestion and context. I looked at this a while back but it wasn't out yet, or there was something blocking it. I had to use a beta version to get version 4 of the html-webpack-plugin, but it seems to work fine. Here's the commit: https://github.com/dillonkearns/elm-pages/commit/31f23a53c4de8bdf74f210feff0afa63c7ec21f1. And [here's the changelog entry](https://github.com/dillonkearns/elm-pages/blob/master/CHANGELOG-NPM.md#114---2020-01-03).

Let me know how it goes with using this with Node 13!
